integration_tests/test_statistics.py

```diff
@@ -187,6 +189,33 @@ def test_linear_regression():
     assert abs(slope + 0.18514007308160782) < eps
     assert abs(intercept - 25.750304506699152) < eps
 
+def test_mode():
+    a: list[i32]
+    a = [3, 1, 12, 4, 0]
+    i: i32
+    i = mode(a)
+    assert i == 3
+
+    b: list[i32]
+    b = [4, 2, 4, 4, 2, 3, 5]
+    j: i32
+    j = mode(b)
+    assert j == 4
+
+    c: list[i32]
+    c = [2, 3, 4, 1, 2, 4, 5]
+    k: i32
+    k = mode(c)
+    assert k == 2
+
+    # TODO: dictionary giving incorrect results with negative numbers
+    # d: list[i32]
+    # d = [-1, 2, -3, -5, -3, -1, 4, -2, 4, -5, -3, 4, -3]
+    # k = mode(d)
+    # print(k)
+    # assert k == -3
```

Comment:
I will look into this.

Reply:
@Smit-create Try using https://github.com/lcompilers/lpython/pull/1090/commits/a8be9663b7cb5d87f23ff33f8a02dc78c179cb3f commit.

Reply:
Thanks, fixed.